packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -32,6 +32,7 @@ import { CreateViewOptions } from '@azure/communication-calling';
 import { DeviceAccess } from '@azure/communication-calling';
 import { DeviceManager } from '@azure/communication-calling';
 import { DominantSpeakersInfo } from '@azure/communication-calling';
+import { DtmfTone as DtmfTone_2 } from '@azure/communication-calling';
```

Comment:
is there a conflicting type name here?

Reply:
No we need to import DtmfTone from Calling but to use it in the components we need to define our own DtmfTone type too. So this is not avoidable

Reply:
I know we did this for `RemoteParticipantState` since there is another type with the same name from calling, but why can't we just use Callings type here? Not blocking but just curious.